### PR TITLE
Add Turkish, Albanian i18n

### PR DESCRIPTION
**Pull request by korabh**
Add Turkish, Albanian file.

### CODE REVIEW

**Comment by korabh** [comment]
@JulienItard :)

**Comment by JulienItard** [comment]
@korabh 👍

File: config/locales/tr.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+en:
```
Comment (JulienItard) [inline config/locales/tr.yml]: wrong

File: config/locales/sq.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
```
Comment (JulienItard) [inline config/locales/sq.yml]: remove line please

**Comment by JulienItard** [comment]
@korabh sorry i merged too fast ... I revert. Could you recreate a new PR with fixes ? Thanks

**Pull request merged**